graylog2-server/src/main/java/org/graylog2/database/MongoConnection.java

```diff
@@ -38,4 +39,11 @@
      * @return The configured MongoDB database.
      */
     MongoDatabase getMongoDatabase();
+
+    /**
+     * Get the client for the configured MongoDB database.
+     *
+     * @return the MongoDB client
+     */
+    MongoClient getMongoClient();
```

Comment:
So far we're not using this anywhere. Are you sure we're going to need it?

Reply:
Good catch! It might be a leftover from a previous iteration of the PR. I will check.

Reply:
Aye, it's not needed anymore and I removed it. Thank you! :+1: